src/components/TextInput/baseTextInputPropTypes.js

```diff
@@ -47,7 +47,10 @@ const propTypes = {
     hideFocusedState: PropTypes.bool,
 
     /** Forward the inner ref */
-    innerRef: PropTypes.func,
+    innerRef: PropTypes.oneOfType([
```

Comment:
Took this from here https://github.com/Expensify/App/pull/17472#issuecomment-1510965317 thanks @eVoloshchak !

And it got rid of the console warning I was getting. I tested that it didn't break the sign-in page either which happened the last time I tried to solve this so think it's working 🙏

Reply:
I'm guessing this should be fixed by the change to `Proptypes.object`, let me see if I can get ios working locally.

Reply:
Has this been resolved?